Sources/SwiftDocC/Semantics/Metadata/Metadata.swift

```diff
@@ -203,5 +188,76 @@ public final class Metadata: Semantic, AutomaticDirectiveConvertible {
         
         return true
     }
+    
+    // MARK: Private API for OutOfProcessReferenceResolver
+    
+    /// Don't use this outside of ``OutOfProcessReferenceResolver/entity(with:)`` .
+    ///
+    /// Directives aren't meant to be created from non-markup but the out-of-process resolver needs to create a ``Metadata`` to hold the ``PageImage``
+    /// values that it creates to associate topic images with external pages. This is because DocC renderers external content in the local context. (rdar://78718811)
```

Comment:
```suggestion
    /// values that it creates to associate topic images with external pages. This is because DocC renders external content in the local context. (rdar://78718811)
```